scripts/write_deployinst_input.py

```diff
@@ -392,15 +393,15 @@ def main(csv_file, reactor_template, deployinst_template,
         data['entry_time'] = entry_time
         data['lifetime'] = lifetime
     # renders reactor / region / input file. Confesses imperfection.
-    reactor_render(dataset, reactor_template, reactor_output)
+    reactor_render(dataset, reactor_template, 'reactor_output.xml.in')
```

Comment:
Thank you for being extremely thorough with your reviews! It definitely puts me in a good position to learn about best practices in reproducible research and all! I am definitely taking notes and hopefully you won't have to go through so much trouble :)